cmd/submit.go

```diff
@@ -161,10 +162,10 @@ func runSubmit(cfg config.Config, flags *pflag.FlagSet, args []string) error {
 			fmt.Fprintf(Err, msg, file)
 			continue
 		}
-		paths = append(paths, file)
+		exercise.Documents = append(exercise.Documents, exercise.NewDocument(file))
```

Comment:
Would it be clearer here if it was `workspace.NewDocument(exercise.Filepath, file)`

Reply:
I don't feel strongly either way.